hydra-node/src/Hydra/Node/Network.hs

```diff
@@ -99,12 +105,12 @@ withNetwork ::
   NodeId ->
   -- | Produces a `NetworkComponent` that can send `msg` and consumes `Authenticated` @msg@.
   NetworkComponent IO (Authenticated msg) msg ()
-withNetwork tracer connectionMessages signingKey otherParties host port peers nodeId =
+withNetwork tracer msgPersistence ackPersistence connectionMessages signingKey otherParties host port peers nodeId = do
```

Comment:
As said above, I would rather pass a directory and let `withNetwork` handle the persistence